teps/0086-changing-the-way-result-parameters-are-stored.md

```diff
@@ -340,11 +342,20 @@ Questions:
 
 ### Store results on PVCs
 
-  - Use PVCs to store results
+  - Use PVCs to store results. The Tekton controller will auto provision the PVC and mount on the task if necessary.
+    The result location will be a pvc/storage path in the pod spec. Then, configure Tekton entrypoint script to retrieve result
+    file contents in the PVCs and replace the pvc/storage path with the new contents. This can avoid the 1.5 MB CRD size as the result contents are
+    replaced during init-container, so it won't be part of the pod or taskrun spec. In the API Spec, we could let users to choose whether to opt-in this
+    feature for each task to reduce the unnecessary overheads if not needed.
+
+Pros:
+- PVCs can leverage with CSI driver to store files in most storages.
+- Able to avoid the 1.5 MB CRD size since the result contents are populated during the init-container.
+- Users can opt-in only for the large result files to be stored in PVCs and keep the rest to use termination logs.
```

Comment:
to use termination logs or any other solution that we make default in future. The sidecar driven approach can be made default if its implementable.